scapy/layers/dot11.py

```diff
@@ -640,11 +645,19 @@ def network_stats(self):
             elif isinstance(p, Dot11EltRates):
                 summary["rates"] = p.rates
             elif isinstance(p, Dot11EltRSN):
-                crypto.add("WPA2")
+                if hasattr(p, "akm_suites"):
+                    auth = akmsuite_types.get(p.akm_suites[0].suite)
+                    crypto.add("WPA2/%s" % (auth))
```

Comment:
The parenthesis are redundant

Reply:
This is still not fixed :/

Reply:
I have replaced the two places with `if p.akm_suites:`, have I misunderstand something?
I have no idea why one check falls, I open another #1917, but still meet fails.